R/derive_param_confirmed_bor.R

```diff
@@ -0,0 +1,528 @@
+#'   numeric values, or `NA`.
+#'
+#' @param subject_keys Variables to uniquely identify a subject
+#'
+#'   A list of symbols created using `vars()` is expected.
+#'
+#' @details
+#'
+#'   1. The input dataset (`dataset`) is restricted to the observations matching
+#'   `filter_source` and to observations before or at the date specified by
+#'   `source_pd`.
+#'
+#'   1. The following potential confirmed responses are select from the restricted input dataset:
+#'
+#'       - `"CR"`: An assessment is considered as complete response (CR) if
+#'           - `AVALC == "CR"`,
+#'           - there is a confirmatory assessment with `AVALC == "CR"` at least
+#'           `ref_confirm` days after the assessment,
+#'           - all assessments between the assessment and the confirmatory
+#'           assessment are `"CR"` or `"NE"`, and
+#'           - there are at most `max_nr_ne` `"NE"` assessments between the
+#'           assessment and the confirmatory assessment.
+#'
+#'       - `"PR"`: An assessment is considered as partial response (PR) if
+#'           - `AVALC == "PR"`,
+#'           - there is a confirmatory assessment with `AVALC %in% c("CR",
+#'           "PR")` at least `ref_confirm` days after the assessment,
+#'           - all assessments between the assessment and the confirmatory
+#'           assessment are `"CR"`, `"PR"`, or `"NE"`,
+#'           - there is no `"PR"` assessment after a `"CR"` assessment in the
+#'           confirmation period, and
+#'           - there are at most `max_nr_ne` `"NE"` assessments between the
+#'           assessment and the confirmatory assessment.
+#'
+#'           If the `accept_sd` argument is set to `TRUE`, one `"SD"` assessment
+#'           in the confirmation period is accepted.
+#'
+#'        - `"SD"`: An assessment is considered as stable disease (SD) if
+#'          - `AVALC %in% c("CR", "PR", "SD")` and
+#'          - the assessment is at least `ref_start_window` days after
+#'          `reference_date`.
+#'
+#'        - `"NON-CR/NON-PD"`: An assessment is considered as NON-CR/NON-PD if
+#'          - `AVALC = "NON-CR/NON-PD"` and
+#'          - the assessment is at least `ref_start_window` days after
+#'          `reference_date`.
+#'
+#'        - `"PD"`: An assessment is considered as progressive disease (PD) if
+#'        `AVALC == "PD"`.
+#'
+#'        - `"NE"`: An assessment is considered as not estimable (NE) if
+#'            - `AVALC == "NE"` or
+#'            - `AVALC %in% c("CR", "PR", "SD")` and the assessment is less than
+#'            `ref_start_window` days after `reference_date`.
+#'
+#'        - `"ND"`: An assessment is considered as not done (ND) if `AVALC ==
+#'        "ND"`.
+#'
+#'        - `"MISSING"`: An assessment is considered as missing (MISSING) if a
+#'        subject has no observation in the input dataset.
+#'
+#'            If the `missing_as_ne` argument is set to `TRUE`, `AVALC` is set to
+#'            `"NE"` for missing assessments.
```

Comment:
i thought this was only "for subjects without an assessment in the input dataset" from your argument description for `missing_as_ne` above?

Reply:
Updated, is it clearer now?